src/components/ContactCustomerSupportButton/index.jsx

```diff
@@ -30,7 +31,13 @@ ContactCustomerSupportButton.propTypes = {
 };
 
 ContactCustomerSupportButton.defaultProps = {
-  children: 'Contact support',
+  children: (
+    <FormattedMessage
+      id="admin.portal.contact.support.button.label"
+      defaultMessage="Contact support"
+      description="Label for the 'Contact support' button in the admin portal."
+    />
+  ),
```

Comment:
@jajjibhai008 we can't use `intl.formatMessage`outside a functional component.
Additionally, If an undefined value is passed as the children prop, it will render "Contact support" as the button text since it's the default value.